File: src/main/java/com/team573/gongguri/domain/groupPurchase/dto/GroupPurchaseWithReviewedResponseDto.java
package com.team573.gongguri.domain.groupPurchase.dto;

import lombok.Builder;

@Builder
public record GroupPurchaseWithReviewedResponseDto(
    Long id,
    String title,
    int maxParticipants,
    Long participantCount,
    Boolean isReviewed,
    String imageUrl,
    int price
) {

}

File: src/main/java/com/team573/gongguri/domain/grouppurchase/dto/GroupPurchaseWithReviewedResponseDto.kt
package com.team573.gongguri.domain.grouppurchase.dto

data class GroupPurchaseWithReviewedResponseDto(
    val id: Long,
    val title: String,
    val maxParticipants: Int,
    val participantCount: Long,
    val isReviewed: Boolean,
    val imageUrl: String,
    val price: Int
)
